readme.md

```diff
@@ -28,8 +28,12 @@ $ boxen --help
     --padding           Space between the text and box border
     --margin            Space around the box
     --center            Center the box
-    --align             Align the text [left|center|right] (Default: left)
-    --width             Set a fixed width for the box
+	  --align, -a         Align the text [left|center|right] (Default: left)
+	  --title, -t         Display a title at the top of the box.
+	  --title-alignment   Align the title in the top bar [left|center|right].
+	  --width, -w         Set a fixed width for the box
+	  --height, -h        Set a fixed height for the box
+	  --fullscreen, -f    Fit all available space within the terminal.
```

Comment:
Incorrect indentation. Look at the existing one.

Reply:
The indentation is exactly the same (One Tab + two spaces). I used copy+paste from the existing ones. It really appears the same both in the code and in the console.